src/main/java/sixgaezzang/sidepeek/projects/exception/message/FileErrorMessage.java

```diff
@@ -0,0 +1,12 @@
+package sixgaezzang.sidepeek.projects.exception.message;
+
+import static sixgaezzang.sidepeek.common.util.CommonConstant.MAX_TEXT_LENGTH;
+import static sixgaezzang.sidepeek.projects.util.ProjectConstant.MAX_OVERVIEW_IMAGE_COUNT;
+
+public class FileErrorMessage {
```

Comment:
```suggestion
public class FileErrorMessage {

    private FileErrorMessage() {}
  
```

`ErrorMessage` 관련 클래스들은 private 생성자를 사용해서 생성을 막으면 좋을 것 같아요!

FileErrorMessage, MemberErrorMessage 등등 적용해주시면 좋을 것 같아요!

Reply:
오오 감사합니다! 생각을 못했네요!!! validator도 다 static이라서 추후에 요것도 바꿔보겠습니다! 우선 예외 메시지, 상수 클래스 적용 완료 햇습니다!